README.md

```diff
@@ -36,7 +36,28 @@ Certain properties of a resource are _semantic_ and have special meaning when us
 * **`deprecatedProperties`**: A property in the `deprecatedProperties` is not guaranteed to be present in the response from a **READ** request. These fields will still be accepted as input to **CREATE** and **UPDATE** requests however they may be ignored, or converted to new API forms when outbound service calls are made.
 * **`replacementStrategy`**: As mentioned above, changing a `createOnlyProperty` requires replacement of the resource by creating a new one and deleting the old one. The default CloudFormation replacement behavior is to create a new resource first, then delete the old resource, so as to avoid any downtime. However, some resources are singleton resources, meaning that only one can exist at a time. In this case, it is not possible to create a second resource first, so CloudFormation must Delete first and then Create. Specify either `create_then_delete` or `delete_then_create`. Default value is `create_then_delete`
-* **`taggable`**: A boolean type property which defaults to true, indicating this resource type supports updatable [`tagging property`](https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html). Otherwise, it indicates this resource type does not contain any updatable [`tagging properties`](https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html).
+* **`taggable`**: [DEPRECATED] ~~A boolean type property which defaults to true, indicating this resource type supports updatable [`tagging property`](https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html). Otherwise, it indicates this resource type does not contain any updatable [`tagging properties`](https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html)~~.
+* **`tagging`**: An object type property that indicates whether this resource type supports tagging, tagging behavior, and what property is used to set tags:
+    * `taggable`: A boolean flag indicating whether this resource type supports tagging. **required**
+    * `tagOnCreate`: A boolean flag indicating whether this resource type supports tagging resources upon creation.
+    * `tagUpdatable`: A boolean flag indicating whether this resource type supports updatable tagging.
+    * `cloudFormationSystemTags`: A boolean flag indicating whether this resource type supports CloudFormation system tags.
```

Comment:
we may want to explain these a little bit more.  I'll let you give a shot at phrasing this for these doc purposes, but here is what I think is missing:

Q: What do we mean by tagging resources on creation?
A: You can pass the tags into the create API, allowing for enforcement of [tags based access control](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_tags.html)

Q: What is "updatable tagging"?
A: You can modify the tags using the update handler.  Now that I'm thinking about this, this property is unnecessary.  We can just check if the `tagProperty` is in `createOnlyProperties`

Q: What does "supports CloudFormation System Tags" mean?
A: The create handler will apply aws prefixed tags as described [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-resource-tags.html)

Reply:
Wording updated.